### ISSUE TITLE
OS X Yosemite dyld lazy symbol binding failed: symbol not found

### ISSUE THREAD

**Issue by nruth** [postmodern/ffi-hunspell#10]
Posted at http://stackoverflow.com/questions/26490964/os-x-yosemite-ffi-hunspell-dyld-lazy-symbol-binding-failed-symbol-not-found in case it's not a gem issue

Since upgrading I've been getting this error when running my tests:

```
dyld: lazy symbol binding failed: Symbol not found: __ZN8HunspellC2EPKcS1_S1_
  Referenced from: /usr/lib/libhunspell-1.2.dylib
  Expected in: flat namespace

dyld: Symbol not found: __ZN8HunspellC2EPKcS1_S1_
  Referenced from: /usr/lib/libhunspell-1.2.dylib
  Expected in: flat namespace
```

Is there something in the gem related to this, or is it a system specific problem? Has anyone else encountered this or fixed it?

**Comment by jordanstephens** [comment]
I ran into this problem as well and solved it by pointing my Gemfile at this github repo. 
```
gem "ffi-hunspell", github: "postmodern/ffi-hunspell"
```
It looks like this was fixed with [this commit (9151663)](https://github.com/postmodern/ffi-hunspell/commit/91516637fdff9cef9bae66aefdd89e1b4a8b5831), but the update has not been pushed out to rubygems. Any particular reason why not? That commit was about ten months ago.

You shouldn't need to run a wide-open `brew upgrade`. That's super heavy-handed.

**Comment by jordanstephens** [comment]
@postmodern it's been about 15 months since this was fixed in [9151663](https://github.com/postmodern/ffi-hunspell/commit/91516637fdff9cef9bae66aefdd89e1b4a8b5831), and it still has not made it into a distributed release.

People are doing crazy things like `brew update && brew upgrade` trying to work around this, not realizing that it was fixed but never released.

Do you have any plans to push an updated release to rubygems so we don't have to continue bundling from the github repo? Is there any reason not to?

**Comment by postmodern** [comment]
Oops, this slipped through the cracks. Released 0.3.1 after manually testing since TravisCI is lagging.

**Comment by jordanstephens** [comment]
@postmodern Thank you, thank you! 🚀

**Comment by matthewbennink** [comment]
I got a similar error message, but then realized that homebrew installed version 0.4.1 of hunspell.  Noting this here in case anyone else experiences the same issue.

```
➜  ~ brew info hunspell
hunspell: stable 1.4.1 (bottled)
Spell checker and morphological analyzer
```

**Comment by willmoorefyi** [comment]
I've opened a PR to fix the 1.4.1 issue above - https://github.com/postmodern/ffi-hunspell/pull/14

All the tests run fine and this fixed the issue on my local development environment, and my tests run on a linux jenkins instance still using hunspell 1.3 so hopefully it's a non-controversial change.
